File: Php/framework/framework/web/mvc/smartjsonresult.php
<?php

namespace Smartkernel\Framework\Web\Mvc {

    use Smartkernel\Framework\Web\SmartMimeType;
    use Smartkernel\Framework\Web\Mvc\SmartContentResult;

/*
     * Json结果
     */

    class SmartJsonResult extends SmartContentResult {
        /*
         * 构造函数
         */

        function __construct() {
            parent::__construct();
            $this->ContentType = SmartMimeType::Json;
        }

    }

}

File: Framework/CSharp/Framework/Framework/Web/Mvc/SmartJsonResult.cs
/*
 * 项目：Smartkernel.Framework
 * 作者：曹艳白
 * 邮箱：caoyanbai@139.com
 */
using System.Web.Mvc;

namespace Smartkernel.Framework.Web.Mvc
{
    /// <summary>
    /// Json结果
    /// </summary>
    public class SmartJsonResult : ContentResult
    {
        /// <summary>
        /// 构造函数
        /// </summary>
        public SmartJsonResult()
            : base()
        {
            this.ContentType = SmartMimeType.Json;
        }
    }
}
